packages/driver/test/cypress/integration/e2e/focus_blur_spec.js

```diff
@@ -669,22 +669,24 @@ describe('intercept blur methods correctly', () => {
     cy.wrap($svg).focus().should('have.focus')
   })
 
-  it('focus area', () => {
+  it('focus area', (done) => {
     cy.visit('http://localhost:3500/fixtures/active-elements.html').then(() => {
       cy.$$(`
       <map name="map">
       <area shape="circle" coords="0,0,100"
       href="#"
       target="_blank" alt="area" />
       </map>
-      <img usemap="#map" src="/__cypress/static/favicon.ico" alt="image" />
+      <img id="test-image" usemap="#map" src="/__cypress/static/favicon.ico" alt="image" />
       `).appendTo(cy.$$('body'))
 
-      cy.get('area')
-      // NOTE: wait needed for firefox, otherwise element is not yet ready/loaded
-      .wait(100)
-      .focus()
-      .should('have.focus')
+      cy.$$('#test-image').on('load', () => {
+        cy.get('area')
+        .focus()
+        .should('have.focus')
+
+        done()
```

Comment:
this will call `done()` before the `cy.get` command is executed, so it will always pass as long as the image loads